### PR TITLE
[Plots] Fix Plots breaking on exceptionally large values

### PR DESCRIPTION
**Pull request by jvigliotta**
Related: https://github.com/NASA-AMMOS/openmct-mcws/issues/15

### Describe your changes:

Put a check in to see if the value to be plotted can be handled by a `Float32Array` and if not handling as we handle other non-plottable values.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/plot/configuration/PlotSeries.js [hunk 1]
```diff
@@ -410,7 +414,7 @@ export default class PlotSeries extends Model {
         const currentYVal = this.getYVal(point);
         const lastYVal = this.getYVal(data[insertIndex - 1]);
 
-        if (this.isValueInvalid(currentYVal) && this.isValueInvalid(lastYVal)) {
+        if ((this.isValueInvalid(currentYVal) && this.isValueInvalid(lastYVal))) {
```
Comment (davetsay) [inline src/plugins/plot/configuration/PlotSeries.js:417]: if this passes linting im cool but it might not like the extra parens

**Review by davetsay — APPROVED** [review]
Looks good. I added one comment.

File: src/plugins/plot/configuration/PlotSeries.js (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/plot/configuration/PlotSeries.js:417]: yeah that was leftover, removing

**Review by davetsay — APPROVED** [review]
great work!

**Pull request merged**